api/views.py

```diff
@@ -0,0 +1,59 @@
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework.viewsets import ModelViewSet
+from rest_framework import status
+
+from . models import Category, Job, Application
+from . serializers import CategorySerializer, JobSerializer, ApplicationSerializer
```

Comment:
Please remove unused imports.
```suggestion
from . serializers import JobSerializer, ApplicationSerializer
```

Reply:
Updated.